src/App/Fossa/Compatibility.hs

```diff
@@ -0,0 +1,51 @@
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+
+module App.Fossa.Compatibility
+  ( main,
+    argumentParser,
+    Argument,
+  )
+where
+
+import App.Fossa.EmbeddedBinary (BinaryPaths, toExecutablePath, withCLIv1Binary)
+import Control.Effect.Lift (sendIO)
+import Data.Text (Text, pack)
+import Effect.Exec (CmdFailure(cmdFailureStdout), AllowErr (Never), Command (..), exec, runExecIO, cmdFailureStderr)
+import Path
+import qualified Data.ByteString.Lazy.Char8 as BL
+import Options.Applicative (Parser, argument, help, metavar, str)
+import System.Exit (exitFailure, exitSuccess)
+import Data.Text.Lazy.Encoding
+import Effect.Logger (Pretty(pretty), logInfo, logSticky, Severity(SevInfo), withLogger)
+
+type Argument = Text
+
+argumentParser :: Parser Argument
+argumentParser = pack <$> argument str (metavar "IMAGE" <> help "The image to scan")
+
+main ::
+  [Argument] ->
+  IO ()
+main args = withLogger SevInfo . runExecIO . withCLIv1Binary $ \v1Bin -> do
```

Comment:
Let's use a less ambiguous name, maybe `compatibilityMain`.  If we refactor the CLI side without qualified imports, this will be very confusing.

Reply:
Good call, I was trying to not be redundant calling`Compatibility.compatibilityMain` but I overlooked being explicit with the import.